pandas/_libs/tslibs/period.pyx

```diff
@@ -1209,8 +1209,7 @@ def period_format(int64_t value, int freq, object fmt=None):
         elif freq_group == 4000:  # WK
             left = period_asfreq(value, freq, 6000, 0)
             right = period_asfreq(value, freq, 6000, 1)
-            return '%s/%s' % (period_format(left, 6000),
-                              period_format(right, 6000))
+            return f"(period_format(left, 6000)/period_format(right, 6000))"
```

Comment:
are we calling this function inside a loop?  if so, we may need to be careful about whether cython fstrings differently from % formatting

Reply:
more generally, id prefer to have non-trivial function calls occur outside of f-space.

Reply:
the CI failures look related to this